gce/gce.go

```diff
@@ -688,6 +694,45 @@ func (s *gceOps) SetInstanceGroupSize(instanceGroupID, instanceGroupLocation str
 	return nil
 }
 
+func (s *gceOps) GetInstanceGroupSize(instanceGroupName string) (int64, error) {
+
+	zonalCluster, err := isZonalCluster(s.inst.clusterLocation)
+	if err != nil {
+		return 0, err
+	}
+
+	var nodePool *container.NodePool
+	if zonalCluster {
+		nodePool, err = s.containerService.Projects.Zones.Clusters.NodePools.Get(
+			s.inst.project, s.inst.clusterLocation, s.inst.clusterName, instanceGroupName).Do()
+	} else {
+		nodePoolPath := fmt.Sprintf("projects/%s/locations/%s/clusters/%s/nodePools/%s",
+			s.inst.project, s.inst.clusterLocation, s.inst.clusterLocation, instanceGroupName)
+		nodePool, err = s.containerService.Projects.Locations.Clusters.NodePools.Get(nodePoolPath).Do()
+	}
+
+	if err != nil {
+		return 0, err
+	}
+
+	nodeCount := int64(0)
+	for _, instanceGroupURL := range nodePool.InstanceGroupUrls {
+
+		nodeGrpName := strings.TrimSpace(filepath.Base(instanceGroupURL))
+
+		temp := strings.SplitAfter(instanceGroupURL, "zones")[1]
```

Comment:
do a check here for the size of the slice after split. For any reason, if zones is not present in the instanceGroupURL, this will panic.